src/base/emu-i386/kvm.c

```diff
@@ -576,10 +563,7 @@ void mmap_kvm(int cap, void *addr, size_t mapsize, int protect, dosaddr_t targ)
 
   assert(cap & (MAPPING_INIT_LOWRAM|MAPPING_KVM));
   /* with KVM we need to manually remove/shrink existing mappings */
-  if (cap & MAPPING_IMMEDIATE)
-    do_munmap_kvm(targ, mapsize);
-  else
-    check_overlap_kvm(targ, mapsize);
+  do_munmap_kvm(targ, mapsize, cap & MAPPING_IMMEDIATE);
```

Comment:
IIRC you removed all users of
MAPPING_IMMEDIATE, so where
is that still set?
We don't use it because we can
mmap behind kvm's back. I don't
see how KVM_VGA_BASE would
change that.